en/orm/validation.rst

```diff
@@ -429,6 +429,29 @@ Note that ``validCount`` returns ``false`` if the property is not countable or d
     // The save operation will fail if tags is null.
     $rules->add($rules->validCount('tags', 0, '<=', 'You must not have any tags'));
 
+Association Link Constraint Rule
+--------------------------------
+
+The ``LinkConstraint`` lets you emulate SQL constraints in databases that don't
+support them. This rule enables you to check if an association does or does not
```

Comment:
I'd say it's useful for databases that do support constraints too, it's easier to provide a nice user experience with rules than having to deal with exceptions.